specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/virtualNetworkGateway.json

```diff
@@ -4284,8 +4284,7 @@
         "ECP256",
         "ECP384",
         "PFS24",
-        "PFS14",
-        "PFSMM"
```

Comment:
What's the impact by removing this in a released version?

Reply:
This is an unsupported field and is not present in both Azure Powershell and Portal. There is only 1 customer who uses this and we are planning to send a comms to customer once this is removed.

Reply:
@gary-x-li Can you please help in reviewing/approving? We have been waiting to send a comms to customer after this deprecated field has been removed.